mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/MapboxDirections.java

```diff
@@ -464,6 +465,18 @@ public MapboxDirections build() throws ServicesException {
           "You should provide at least two coordinates (from/to).");
       }
 
+
+      if (profile != null && profile.equals(DirectionsCriteria.PROFILE_DRIVING_TRAFFIC)
```

Comment:
@srabenja Considering that `profile` and `coordinates` are required parameters, how about we add a check before these two raise an exception if either is `null` (or empty)?